R/header.R

```diff
@@ -0,0 +1,36 @@
+#' DfE header banner
+#'
+#' @description
+#' This function uses `shinyGovstyle::header()` to create a header banner
+#' using the DfE logo.
+#'
+#' @param secondary_text Secondary header to supplement the main text
+#' @param secondary_link Add a link for clicking on secondary header
+#' @return a header html shiny object
+#'
+#' @seealso [shinyGovstyle::header()]
+#' @export
+#' @examples
+#'
+#' if (interactive()) {
+#'   ui <- fluidPage(
+#'     dfeshiny::header(
+#'       secondary_text = "User Examples"
+#'     )
+#'   )
+#'
+#'   server <- function(input, output, session) {}
+#'
+#'   shinyApp(ui = ui, server = server)
+#' }
+header <- function(secondary_text,
```

Comment:
I think let's change the param names as they don't make sense in the context of this particular function. I think people shouldn't need to know about `shinyGovstyle::header()` for the param names to make sense. I'd go for dashboard_title in place of secondary text. And I'd be tempted to just drop secondary link as a parameter here as I can't think of any reason for the dashboard title link taking them anywhere except the dashboard front page.